trade/api/binance_api.js

```diff
@@ -144,34 +122,34 @@ module.exports = function() {
    * @param {String} method Request type: get, post, delete
    * @returns {*}
    */
-  function protectedRequest(path, data, type) {
-    let url = `${WEB_BASE}${path}`;
-    const urlBase = url;
+  function protectedRequest(type, path, data) {
+    const url = `${WEB_BASE}${path}`;
 
     data.timestamp = Date.now();
     data.signature = getSignature(config.secret_key, getParamsString(data));
 
     const bodyString = getParamsString(data);
 
-    if (type !== 'post') {
-      url = getUrlWithParams(url, data);
-    }
-
     return new Promise((resolve, reject) => {
       const httpOptions = {
-        url: url,
+        url,
         method: type,
         timeout: 10000,
```

Comment:
Try set `timeout: 1` and run `/balances` command.

Expected:
`Request to https://api.binance.com/api/v3/account with data timestamp=1681096444784&signature=49e893465635be06e99058ff6991e97fb421a23ddeaa1d53c5ff1f3ccc48bd89 failed. Unknown error: AxiosError: timeout of 1ms exceeded. Rejecting…`

In fact: `Error while processing response of request to https://api.binance.com/api/v3/account with data timestamp=1681097715028&signature=e076850fa6286e0c5c523700ad34dc1dd2fbcb4e34bc9f7812c9ebaec755c47a: TypeError: Cannot set properties of undefined (setting 'binanceErrorInfo'). Data object I've got: undefined.`